src/toil_scripts/tools/preprocessing.py

```diff
@@ -43,3 +43,42 @@ def cutadapt(job, r1_id, r2_id, fwd_3pr_adapter, rev_3pr_adapter):
         r1_cut_id = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'R1_cutadapt.fastq'))
         r2_cut_id = None
     return r1_cut_id, r2_cut_id
+
+
+def run_bwa_index(job, ref_id):
```

Comment:
Neither of these two functions are what I would call preprocessing. Maybe create an `indexing` module?